application-form/src/components/share/multiForm/components/share/InputText.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="input-item" v-if="isShow">
+    <div class="input-title">
+      <InputRequire v-if="requireItem" />
+      <div class="input-name">{{ title }}</div>
+    </div>
+    <input
+      :type="type"
+      @keyup="onChange(keyword)"
+      :class="{
+        error: errorMsg,
+      }"
+      v-model="keyword"
+      :readonly="isAdminPage"
```

Comment:
không ảnh hưởng đến logic nhưng chỗ này nên sắp xếp cho đẹp gọn từ trên xuống dưới:
1. các directives như: v-if v-for ...
2. các prop không có dấu : đằng trước, ví dụ như: placeholder="Nhập từ khóa"
3. các prop có dấu : đằng trước
4. các event @eventName